test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,38 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env sets docker environment variables for supported shells.
+INFO: This feature runs against shell instance. To use non-default shell, please select
+one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_SHELL parameter of make integration.
+
+  Scenario: Docker client is not set up
+  INFO: This scenario starts interactive shell instance, which will be closed in the end of this feature.
+    Given user starts shell instance
+     When executing in shell "docker info"
+     Then stdout of shell should not contain "Operating System: Minishift Boot2Docker ISO Version"
+      And stdout of shell should not contain "Name: minishift"
+
+  Scenario: Starting minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift has state "Running"
+
+  Scenario: Setting Docker client using docker-env command
+     When executing in shell "minishift docker-env" succeeds
+      And executing in shell shell's stdout of previous command line by line
```

Comment:
this does the eval of `docker-env`... can we also call it something like `eval` instead?

Reply:
@gbraad calling it eval is shell-centric I think, I have tried to describe the actual process, however I agree that this sentence is a little bit complicated.